### ISSUE TITLE
TwilioVideoParticipantView sizing issue on Android

### ISSUE THREAD

**Issue by natashamcquillan** [blackuy/react-native-twilio-video-webrtc#616]
### Steps to reproduce
1. Use TwilioVideoParticipantView with scaleType 'fit'
2. Style so that this component will take up 100% of the height of it's container
3. Run on Android
4. Share a screen / use a camera with portrait dimensions

### Expected behaviour
TwilioVideoParticiapantView will take up the full height of the container, with the full view of the camera/ screenshare

### Actual behaviour
The component it's self take up the full height of the container but the child Video component does not. It will take up the full width but only 50% of the height, cutting off parts of the camera / screenshare

### Environment
- **Node.js version**: 17.0.1
- **React Native version**: 0.64.2
- **React Native platform + platform version**: Android 11.0 (R) - API 30

### react-native-twilio-video-webrtc
**Version**: npm version

**Comment by HimikoT-T** [comment]
Any solutions?

**Comment by natashamcquillan** [comment]
It's not a perfect solution, it's what I've got in place for now:
- on IOS keep the scaleType as fit
- on Android change the scaleType to fill
It's not perfect as the sides of any landscape video feeds are being clipped on android

**Comment by HimikoT-T** [comment]
yep, this is my problem right now, just like this one https://github.com/blackuy/react-native-twilio-video-webrtc/issues/436#issue-781334269 but this was on LocalVideo, ours is on the TwilioVideoParticipantView, I hope @slycoder has a solution on this one

**Comment by HimikoT-T** [comment]
@natashamcquillan Hey I have a good news just add this to your fork e9b5ef293507a5bde4876679dbd55699eda24a81 and sizing issue will be solved
